dvc.yaml

```diff
@@ -1,47 +1,29 @@
 stages:
   polemo2_preprocess_dataset:
-    cmd: PYTHONPATH=. python3 experiments/scripts/preprocess_dataset.py --ds polemo2 --cfg-type lightning
+    cmd: PYTHONPATH=. python3 klajster/experiments/scripts/preprocess_dataset.py --ds polemo2 --cfg-type lightning
```

Comment:
Czemu to jest w katalogu `klajster`? IMO bym to zostawił w katalogu `experiments`